services/gundeck/src/Gundeck/Push/Native.hs

```diff
@@ -64,7 +64,6 @@ push1 m a = do
     -- * Others use the default level of the specific platforms, which is 4 weeks
     --   for both APNS* and GCM
     ttl d
-        | msgTransient m = Just (Aws.Seconds 0)
```

Comment:
`Gundeck.Push.Native.Types.msgTransient` always returns false, since we only have `Notice` `Message`s.  So this condition will never fire and the line can be removed.

I admit that I am reasoning from an entirely syntactic perspective, though.  Perhaps the underlying logic is more complicated than the grammar here?

Reply:
> Gundeck.Push.Native.Types.msgTransient always returns false

I believe this to be a bug in the current code. The way `Notice` notifications were used in the past (only as a fallback in case Cipher/Plaintext notifications could not be delivered) is different to the way `Notice` notifications are used now (i.e. always). 

I might be wrong - in that case, please explain *why* `Notice` notifications should not distinguish transientness and why they should be sent to APNS/FCM with a 4 week validity ("but the current code does that" is not an acceptable answer).